### PR TITLE
test bull library

### PR DESCRIPTION
**Pull request by GuohaoOuyang**
Hi Jess, check it out.

### CODE REVIEW

File: server/app.js [hunk 1]
```diff
@@ -50,4 +50,52 @@ app.use(notFound);
...
+// test bull
+var Queue = require("bull");
+
+const myFirstQueue = new Queue("my-first-queue", {
+  redis: { port: 6379, host: "127.0.0.1" },
+});
+
+// job producers
+const job = myFirstQueue.add({ foo: "bar" });
+// myFirstQueue.add({ video: "http://example.com/video1.mov" });
```
Comment (jessdesr) [inline server/app.js]: You can get rid of the example code

File: server/app.js [hunk 2]
```diff
@@ -50,4 +50,52 @@ app.use(notFound);
...
+
+client.on("ready", function () {
+  console.log("redis is running");
+});
+
+// test bull
+var Queue = require("bull");
+
+const myFirstQueue = new Queue("my-first-queue", {
+  redis: { port: 6379, host: "127.0.0.1" },
```
Comment (jessdesr) [inline server/app.js]: you can probably put these values into the environment file

Comment (GuohaoOuyang) [reply server/app.js]: Yeah, you right boss.🙂

**Review by jessdesr — APPROVED** [review]
Looks good as first implementation, this will need to be built upon once we're actually automating the scraping tasks

**Pull request merged**